### ISSUE TITLE
capture() fn returning none is some mac

### ISSUE THREAD

**Issue by rapidshaurya** [nashaofu/screenshots-rs#29]
My friend uses this crate for taking screenshots in Mac OS X (version 12.2.1), but after the Mac OS X update (version 13.0.1),
In capture fn, it returns null. Is this crate dependent on any driver or software on the Mac?

**Comment by arividar** [comment]
I have this issue an a Mac M1 - can you show how you fixed it?

**Comment by nashaofu** [comment]
Is it the same question as this? #38
I can't reproduce this problem on my Intel chip Mac.

**Comment by rapidshaurya** [comment]
I can help you with this issue.
this issue is related to core graphic fn which you use on the crate
instead of using image fn use this


let cg_display = CGDisplay::new(display_info.id);
    let cg_image = CGDisplay::screenshot(
        cg_display.bounds(),
        kCGWindowListOptionOnScreenOnly,
        kCGNullWindowID,
        kCGWindowImageDefault,
    )?;
This will solve all your mac related problems
both for m1 and intel chips.

**Comment by nashaofu** [comment]
I have found the problem，The following code can be reproduced. I have fixed this problem in this #41 
```rust
  let mut cg_rect = cg_display.bounds();
  let origin = cg_rect.origin;
  // Set size is very small
  cg_rect.size = CGSize::new(2.0, 2.0);

  let cg_image = CGDisplay::screenshot(
    cg_rect,
    kCGWindowListOptionOnScreenOnly,
    kCGNullWindowID,
    kCGWindowImageDefault,
  )?;

  // bytes length Is greater than width * height * 4
  assert_eq!(cg_image.data().bytes().len(), cg_image.width() as * cg_image.height() * 4)
```

**Comment by nashaofu** [comment]
fixed #41
